lcUILua/createActions/arcoperations.lua

```diff
@@ -19,11 +19,19 @@ function ArcOperations:_init(id)
     self.Arc_SecondPoint = nil
     self.Arc_ThirdPoint = nil
     self.Arc_Center = nil
+end
+
+function ArcOperations:_init_default()
```

Comment:
In this function (and every other constructor), you should assign the variable `self.step`.
This variable contains the name of the function that will be called on the next event. By default, it is `"ArcWith3Points"`, as declared in the constructor line 15.